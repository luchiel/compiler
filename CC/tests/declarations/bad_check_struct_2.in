int main()
{
struct { int b; } a;
a.c = 5;
}