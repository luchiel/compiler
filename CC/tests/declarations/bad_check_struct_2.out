Error: UndefinedSymbol:4:3 c
